cmd/kubeadm/app/preflight/checks_test.go

```diff
@@ -308,6 +309,222 @@ func TestRunJoinNodeChecks(t *testing.T) {
 	}
 }
 
+func TestFileExistingCheck(t *testing.T) {
+	f, err := ioutil.TempFile("/tmp", "exist")
```

Comment:
It seems possible to use `os.CreateTemp` instead

Reply:
Could I ask a question humbly, Where is his advantage?

Reply:
https://golang.org/doc/go1.16#ioutil

It is recommended to be migrated